src/syslogd/arguments.c

```diff
@@ -141,6 +141,9 @@ void parse_arguments(int argc, char **argv, struct globals *g)
 			case 'a':
 				set_input(INPUT_UNIX, optarg, -1);
 				break;
+			case 'b':
```

Comment:
There is a parameter in the `log_format` configuration file. I think it would be better to add to it the possibility to specify boot-id instead of hardcode of this value in the message line.